packages/@ember/object/tests/create_test.js

```diff
@@ -40,43 +39,43 @@ moduleFor(
     }
 
     ['@test calls computed property setters'](assert) {
-      let MyClass = EmberObject.extend({
-        foo: computed({
-          get() {
-            return "this is not the value you're looking for";
-          },
-          set(key, value) {
-            return value;
-          },
-        }),
-      });
+      let MyClass = class extends EmberObject {
+        @computed
+        get foo() {
+          return this._foo;
+        }
+        set foo(value) {
+          this._foo = value;
+        }
+      };
 
       let o = MyClass.create({ foo: 'bar' });
       assert.equal(o.get('foo'), 'bar');
     }
 
-    ['@test sets up mandatory setters for simple properties watched with observers'](assert) {
-      if (DEBUG) {
-        let MyClass = EmberObject.extend({
-          foo: null,
-          bar: null,
-          fooDidChange: observer('foo', function () {}),
-        });
+    // TODO: Determine if there's anything useful to test here with observer helper gone
```

Comment:
We need to decide if this, and subsequent tests, should be updated and kept around.